modules/express/src/expressApp.ts

```diff
@@ -299,9 +303,18 @@ export async function init(): Bluebird<void> {
 
   const server = await createServer(cfg, expressApp);
 
-  const { port, bind } = cfg;
+  const { port, bind, ipc } = cfg;
   const baseUri = createBaseUri(cfg);
 
-  server.listen(port, bind, startup(cfg, baseUri));
+  if (ipc) {
+    if (process.platform !== "win32" && fs.existsSync(ipc) && fs.statSync(ipc).isSocket()) {
+      fs.unlinkSync(ipc);
```

Comment:
I'm curious why we need to call `unlink` here before binding to the IPC socket - is there some magic here?